### ISSUE TITLE
Works on Cassandra database?

### ISSUE THREAD

**Issue by SeungukHan** [oltpbenchmark/oltpbench#124]
Hi folks,

Does OLTP Bench work on Cassandra database?

I can't find any documents about using other databases. (cassandra)

I hope to execute OLTP Bench with TPC-C on cassandra.

Can I have any advice?

**Comment by apavlo** [comment]
Does Cassandra have a JDBC driver? If so, then you can try it out and let us know if it works.

You will need to edit this file and add "CASSANDRA" as a supported `DatabaseType` first:

https://github.com/oltpbenchmark/oltpbench/blob/master/src/com/oltpbenchmark/types/DatabaseType.java#L23

**Comment by SeungukHan** [comment]
Cassandra have JDBC driver but it  doesn't use SQL. Instead, it uses CQL (Cassandra Query  Language). Both are quite similar but  not same.
I  think  extra efforts  will be needed  for  use  cassandra  on OLTP  bench like editting  read,  wirte function in  CQL format.
Are there  any easier way  than  what  I explain above?

Thanks :)

**Comment by apavlo** [comment]
Yes, this is correct. You don't even have to edit the Java code. You can add the Cassandara-specific queries to this file:

https://github.com/oltpbenchmark/oltpbench/blob/master/src/com/oltpbenchmark/benchmarks/tpcc/tpcc-dialects.xml

**Comment by ashrafMahgoub** [comment]
Any luck with getting Cassandra integrated with OLTP ? I am having trouble integrating cassandra's jdbc driver with the tool.
